### PR TITLE
chore: re-add index.js

### PR DESCRIPTION
**Pull request by admbtlr**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+let surveyorId, projectId, xFormId
+
+const setTimeoutPromise = util.promisify(setTimeout)
+
+const log = (logText, logLevel = 0) => {
+  if (logLevel > 0) {
+    const d = new Date()
+    console.log(d.getHours().toString().padStart(2, '0') +
+      ':' + d.getMinutes().toString().padStart(2, '0') +
+      ':' + d.getSeconds().toString().padStart(2, '0') + ' ' + logText)
```
Comment (quesebifurcan) [inline index.js]: would `d.toLocaleTimeString()` or `d.toISOString()` work here?

File: index.js [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+  }
+}
+
+const generateOptions = (module, entity, payload, method, id, pathExtension) => {
```
Comment (quesebifurcan) [inline index.js]: Shadowing `module` built-in here. Can we pick another name for this argument?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  const plural = entity == 'entity' ? 'entities' : `${entity}s`
+  let options = {
+    uri: `${urls[module]}/${plural}/${id ? id + '/' : ''}${pathExtension ? pathExtension + '/' : ''}`,
```
Comment (quesebifurcan) [inline index.js]: Minor nit, but I try to avoid nesting string concatenation inside of template strings.

File: index.js [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+  log('Creating project')
+  const options = generateOptions('odk', 'project', {
+    'name': 'deployment-test',
+    'surveyors': [ surveyorId ]
+  })
+  return rp(options)
+    .then(response => {
+      projectId = response.project_id
+      return projectId
+    })
```
Comment (quesebifurcan) [inline index.js:89]: What happens when this fails? Can/should the script keep running without assigning to projectId? Or is this intended to be caught here https://github.com/eHealthAfrica/aether-deployment-test/blob/a358f5d090bbf5c7ede70172014fecf2a52bbbea/index.js#L240?

File: index.js [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+  })
+  log(totalSubmissions.length + ' submissions')
+  return Promise.all(totalSubmissions)
+}
+
+const tearDown = () => {
+  log('Tear down')
+  return deleteSurveyor()
+    .then(deleteProject)
+    .then(deleteKernelProject)
```
Comment (quesebifurcan) [inline index.js:230]: What happens if one of these fails? Maybe add a catch block with some info so we can clean up manually?

File: index.js [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+  odk: process.env.ODK_URL,
+  kernel: process.env.KERNEL_URL
+}
+
+const tokens = {
+  odk: process.env.ODK_ADMIN_TOKEN,
+  kernel: process.env.KERNEL_ADMIN_TOKEN
+}
+
+let surveyorId, projectId, xFormId
```
Comment (quesebifurcan) [inline index.js]: `xFormId` isn't used anywhere, or?

**Review by quesebifurcan — CHANGES_REQUESTED** [review]
Nice, very useful!

File: index.js (hunk 4 above)
Comment (admbtlr) [reply index.js:89]: Yes, that's the intention - but you're right, I could put in some more granular error messages

File: index.js (hunk 6 above)
Comment (admbtlr) [reply index.js]: True - a relic from a previous version

File: index.js (hunk 3 above)
Comment (admbtlr) [reply index.js]: 👍

File: index.js (hunk 2 above)
Comment (admbtlr) [reply index.js]: 👍

File: index.js (hunk 1 above)
Comment (admbtlr) [reply index.js]: 👍

**Comment by admbtlr** [comment]
@quesebifurcan can you review again please?

**Pull request merged**